README.md

```diff
@@ -1,119 +1,24 @@
-
-```C#
-using System;
-using UnityEngine;
-using UnityEngine.Events;
-using UnityEngine.UI;
-
-public class PurchasesListener : Purchases.UpdatedPurchaserInfoListener
-{
-    void ButtonClicked(string product)
-    {
-        Purchases purchases = GetComponent<Purchases>();
-        purchases.MakePurchase(product, (productIdentifier, purchaserInfo, userCancelled, error) =>
-        {
-            if (!userCancelled)
-            {
-                if (error != null)
-                {
-                    LogError(error);
-                }
-                else
-                {
-                    DisplayPurchaserInfo(purchaserInfo);
-                }
-            } else
-            {
-                Debug.Log("Subtester: User cancelled, don't show an error");
-            }
-        });
-    }
-    
-    void DoOtherStuff()
-    {
-        var purchases = GetComponent<Purchases>();
-        var data = new AdjustData
-        {
-            adid = "test",
-            network = "network",
-            adgroup = "adgroup",
-            campaign = "campaign",
-            creative = "creative",
-            clickLabel = "clickLabel",
-            trackerName = "trackerName",
-            trackerToken = "trackerToken"
-        };
-
-        purchases.AddAttributionData(JsonUtility.ToJson(data), Purchases.AttributionNetwork.ADJUST);
-        
-        purchases.GetPurchaserInfo((info, error) =>
-        {
-            Debug.Log("purchaser info " + info.ActiveSubscriptions);
-            if (error != null) {
-                LogError(error);
-            }
-        });
-        purchases.GetProducts(new []{ "onemonth_freetrial", "annual_freetrial" }, (products, error) =>
-        {
-            Debug.Log("getProducts " + products);
-            if (error != null) {
-                LogError(error);
-            }
-        });
-        
-        Debug.Log("user ID " + purchases.GetAppUserId());
-    }
-
-    void RestoreClicked()
-    {
-        var purchases = GetComponent<Purchases>();
-        purchases.RestoreTransactions((purchaserInfo, error) =>
-        {
-            if (error != null)
-            {
-                LogError(error);
-            }
-            else
-            {
-                DisplayPurchaserInfo(purchaserInfo);
-            }
-        });
-    }
-
-    
-    public override void PurchaserInfoReceived(Purchases.PurchaserInfo purchaserInfo)
-    {
-        DisplayPurchaserInfo(purchaserInfo);
-    }
-    
-    private void logError(Purchases.Error error)
-    {
-        Debug.Log("Subtester: " + JsonUtility.ToJson(error));
-    }
-
-
-    private void DisplayPurchaserInfo(Purchases.PurchaserInfo purchaserInfo)
-    {
-        // Show purchaser info on screen
-    }
-}
-
-```
+## Getting Started
+For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/v3.0/docs).
```

Comment:
could we make this point to https://docs.revenuecat.com/docs/unity instead? that way it takes you straight into unity